### ISSUE TITLE
import style.css/scss in App.js?

### ISSUE THREAD

**Issue by muhrusdi** [alidcastano/rogue.js#66]
can i import style.css/scss in App.js file?

**Comment by alidcastano** [comment]
Are you using Razzle? 

I haven't tried this personally (I use css-in-js) but I don't see why it wouldn't work since razzle does support this

does it not work when you try it?

**Comment by muhrusdi** [comment]
its work in development, but not work in production, the file (bundle.xxx.css) is not loaded in browser.

![screen shot 2018-07-24 at 12 05 03](https://user-images.githubusercontent.com/9457288/43116363-ddcc88f6-8f39-11e8-8ca5-bb977a9b6774.png)

**Comment by alidcastano** [comment]
ahh yea there needs to be to inject the assets script tag

e.g. 
```
const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);

headTags: [<link rel="stylesheet" href={assets.client.css} />]
```

something like this will need to be added

**Comment by muhrusdi** [comment]
undefined css in assets variable, just js.

![screen shot 2018-07-24 at 14 08 19](https://user-images.githubusercontent.com/9457288/43120014-46ae2bde-8f4b-11e8-9e94-cbc32293b2a1.png)

and i add link tag in the file App.js, because there is no headTags in the rouge api
![screen shot 2018-07-24 at 14 10 54](https://user-images.githubusercontent.com/9457288/43120109-9efce636-8f4b-11e8-8ab6-2ebba3b4ad45.png)

**Comment by alidcastano** [comment]
yeah doing it via getInitialProps is a good temporary workaround

hmm is it undefined in production? just check if it exists first before setting it

also just remembered loadable-components (for code splitting) only supports css-in-js (https://github.com/smooth-code/loadable-components/issues/94), so now that Rogue is independent of build setup we might have to also support another library for doing this

**Comment by muhrusdi** [comment]
thanks

**Comment by alidcastano** [comment]
So to confirm, adding headTag works?

**Comment by muhrusdi** [comment]
yes its work, the file css is loaded in prod , undfined in dev. but why this way does not work `const rogue = new Rogue(App, { headTags: [`${css}`], bundleUrl: bundleUrl})`
